### DISCUSSION TITLE
What Version of CUDA Toolkit/cuDNN are Needed to Run the GPU Version of DeePMD-Kit Version 3.0?

### DISCUSSION THREAD

**Discussion by agvent** [deepmodeling/deepmd-kit#4477] [Q&A] [answered]
Hello,

I have successfully installed DeePMD-Kit version 3.0 on my HPC in a virtual environment using the conda-forge channel along with horovod. I would like to use the GPU version of DeePMD-Kit on my HPC. I have the following questions:

1. I noticed that the other options to install DeePMD-Kit in the documentation provide a CPU or GPU version. However, the conda-forge channel only provides one version. Does the conda-forge version automatically install the GPU version? (Note: my HPC has GPUs).
2. From my understanding, I need to install CUDA Toolkit and cuDNN to be able to use the GPU version of DeePMD-Kit. My HPC already has some versions of CUDA toolkit and cuDNN available so I am trying to determine if I can use them. Which version of CUDA toolkit and cuDNN are required to use DeePMD-Kit Version 3.0? Based on the documentation, it appears that CUDA toolkit Version 12 is needed, but I am not completely sure. (Additionally, for reference, I am using the versions of Tensorflow and Pytorch that came installed when I installed DeePMD-Kit Version 3.0 using the conda-forge channel)
3. Are there any other packages I need to install besides CUDA toolkit and cuDNN to use the GPU version of DeePMD-Kit?

Thank you in advance for any help able to be provided, and please let me know if there are any further questions.

**Comment by njzjz** [comment, marked as answer]
> 2\. I need to install CUDA Toolkit and cuDNN

Conda should have installed them. If not, you may check to see if you installed the CPU version.

Ref: https://conda-forge.org/docs/user/tipsandtricks/#installing-cuda-enabled-packages-like-tensorflow-and-pytorch
